docs/developer-guide/rosplane/estimator/estimator-base.md

```diff
@@ -0,0 +1,49 @@
+# Estimator Base
+
+## Overview
+
+The estimator base implements the basic ROS interfaces for the estimator.
+This includes setting up subscribers, publishers and initializing parameter management.
+The idea of the base class, is that all interfacing with ROS and shared resources across all inheritance levels happens or are contained in this class.
+
+## ROS Interfaces
+
+The estimator has the following ROS interfaces.
+
+| ![Diagram of Estimator ROS Sensors](../../../assets/estimator_assets/estimator_ros_input_output.png "Estimator ROS Sensors") |
+|:--:|
+|*Figure 1: Estimator's ROS interactions.*|
+
+The estimator has several ROS interfaces that are tracked as member variables of the `estimator_base` class.
+They are summarized in the table below:
+
+| ROS Interface | Topic | Explanation | Message Type |
+|:------:|:-------:| :---: | :---: |
+| <div style="white-space: nowrap;">`vehicle_state_pub_`<div> | `/estimated_state` | Publishes the estimated state of teh vehicle. | State.msg |
+| <div style="white-space: nowrap;">`gnss_fix_sub_`<div> | `/navsat_compat/fix` | Subcribes to the GNSS position information. | NavSatFix.msg |
+| <div style="white-space: nowrap;">`gnss_vel_sub_`<div> | `/navsat_compat/vel` | Subcribes to the GNSS velocity information. | TwistStamped.msg |
+| <div style="white-space: nowrap;">`imu_sub_`<div> | `/imu/data` | Subcribes to the IMU data (both Gyro and Accel). | Imu.msg |
+| <div style="white-space: nowrap;">`baro_sub_`<div> | `/baro` | Subcribes to the barometer pressure information. | Barometer.msg |
+| <div style="white-space: nowrap;">`airspeed_sub_`<div> | `/airspeed` | Subcribes to the differential pressure information. | Airspeed.msg |
+| <div style="white-space: nowrap;">`status_sub_`<div> | `/status` | Subcribes to the aircraft status information. | Status.msg |
+
+!!! note 
+    The NavSatFix, TwistStamped and Imu messages are from standard ROS message packages. Barometer, Airspeed and Status messages are from `rosflight_msgs`.
```

Comment:
I had that originally, but it was very long and it wrapped a little weird. It could be more clear, but I think it could be harder to understand if it is wrapped strangely, since you would have to look at a different spot for each row since the wrapping is consistent. Also it is a lot of repeated info.

Reply:
Yeah, I was wondering earlier today if we should somehow increase the width of our page... There's quite a bit of white space on the sides. That could help with some table size constraints.

Reply:
Actually I think that if there is a way we should widen it.

Reply:
I am going to add this to the misc improvements and resolve this.